slsim/lsst_science_pipeline.py

```diff
@@ -745,6 +745,106 @@ def dp0_time_series_images_data(butler, center_coord, radius="0.1", band="i", si
     return table_data
 
 
+def opsim_time_series_images_data(
+    butler, center_coord, radius="0.1", band="i", size=101
+):
+    """Creates time series data from opsim database.
+
+    :param butler: butler object
+    :param center_coord: A coordinate point around which we need to create time series
+        images.
+    :param radius: radius for query
+    :param band: imaging band
+    :param size: cutout size of images
+    :return: An astropy table containg time series images and other information
```

Comment:
Can you specify the format a bit more, e.g. what other class needs it and how it should be consistent with it?

Reply:
I think I described the parameters well now